### DISCUSSION TITLE
What is the J5 connector for (under the SD card)? How to add controls for heated chamber?

### DISCUSSION THREAD

**Discussion by kdobrev** [bassamanator/everything-sovol-sv06#18] [Ideas]
Hi,
I am wondering if this empty connector is for another sd card interface or could be for some inputs/outputs.

I saw that pins 13,14 and 15 from the main chip are not used and if we micro-solder to them we can use them for stuff.

But I am not able to determine which pins are these.

**Comment by bassamanator** [comment]
Don't know unfortunately. I did, however, find some pin information [here](https://forum.drucktipps3d.de/forum/thread/21977-pinbelegung-sovol-sv06-board-und-hotend-stecker/). That forum in general has a lot of info. You may have to use Google translate.

**Comment by BeauSlim** [comment]
Do you mean the ICSP 4-pin header between the USB port and the display header?  If so, yes, you can use the 2 middle pins as IO.  I use mine to control 2 fan MOSFET boards.  They are PA13 and PA14 in Klipper.

**Reply by bassamanator** [reply]
Hey man,

Do you mean these 2 pins:
![image](https://github.com/bassamanator/everything-sovol-sv06/assets/61985779/bed11d31-b50c-44a1-9667-cc7658857dc9)

If so, do you mind pasting how they're using in your config?
